### ISSUE TITLE
Analyze popup only triggers once, won't fire again on 2nd page

### ISSUE THREAD

**Issue by ecaron** [ecaron/firefox-font-finder#1]
As reported:
- https://addons.mozilla.org/en-us/firefox/addon/font-finder/reviews/386286/
- https://addons.mozilla.org/en-us/firefox/addon/font-finder/reviews/386540/

**Comment by ricardozea** [comment]
And https://addons.mozilla.org/en-us/firefox/addon/font-finder/reviews/407362/

EDIT-- Added step by step description:

The step by step is quite simple:
1. On any web page, try this one we're on right now, double click on a word to select it
2. Right click on the selected word
3. Click on 'FontFinder' from the context menu
4. Click on 'Analyze selection'
5. The information dialog box 'appears'

If you try it the first time after opening Firefox, no problem, it works just fine.

But after that, following the same steps mentioned above, nothing happens after step #4. In other words the  information dialog box doesn't appear anymore.

## 

Nonetheless, FontFinder seems to be working just fine right now when typing these steps, not sure if you submitted an update or what, but as I'm typing this message and checking the add-on again, it's working just fine.

Note that I decided to re-install the add-on in case a fresh install would fix the problem by overwriting old files or something. However, after re-installing and testing several hours ago, FontFinder did not work.

After re-installing the add-on, I restarted my Firefox as usual. But after that, I have not restarted my Firefox and the add-on seems to be working every time I perform the steps mentioned above.

I checked on a local HTML page and on several other 'online' pages, like this Issues forum, the FontFinder add-on reviews page (https://addons.mozilla.org/en-US/firefox/addon/font-finder/reviews/407362/)

## 

PS. This is more a suggestion than part of the issue discussed here: The Title of the dialog window, try changing `[JavaScript Application]` to something more meaningful like `[FontFinder Firefox add-on: Font properties window]` or similar, but definitely something more meaningful.

It appears that the window is a JavaScript error, or something is wrong with the page one is looking at.

## 

Thanks.

**Comment by andy-portmen** [comment]
@ecaron do you have any clue why this is happening?

**Comment by ecaron** [comment]
I'm thinking we have some variables leaking. That's the only logic I could think that would cause some websites this exhibit this problem but not others.

**Comment by davidbell** [comment]
I was having the same issue here, it used to work fine, then one day it started not appearing after the first time. Restarting Firefox made it work again, but again only once.

I found this page after getting frustrated with trying to work out what part of a font-stack was being rendered on a very basic page I was making (plain HTML, no JavaScript, very little content & styling). Following ricardozea mentioning re-installing the plug-in I tried that and now it seems to be working fine.
